### PR TITLE
Add explanation for unexpected error msg

### PR DESCRIPTION
**Pull request by HackToday**
Fixes: #18698
Signed-off-by: Kai Qiang Wu(Kennan) <wkqwu@cn.ibm.com>

### CODE REVIEW

**Comment by HackToday** [comment]
ping @thaJeztah 
Could you check if such is OK?

**Comment by HackToday** [comment]
@thaJeztah  not sure who would maintain this opensource repo. Is it ok to be merged ?

**Comment by calavera** [comment]
LGTM

**Comment by HackToday** [comment]
@thaJeztah  PTAL. Thanks  Or some other?

**Comment by HackToday** [comment]
ping @moxiegirl

File: docs/project/set-up-dev-env.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ To remove unnecessary artifacts:
     This command uses `docker images` to list all images (`-a` flag) by numeric
     IDs (`-q` flag) and filter them to find dangling images (`-f dangling=true`).
     Then, the `docker rmi` command forcibly (`-f` flag) removes
-    the resulting list. To remove just one image, use the `docker rmi ID`
-    command.
-
+    the resulting list. If you get a "docker: "rmi" requires a minimum of 1 argument."
+    message, that there were no dangling images. To remove just one image, use the
```
Comment (thaJeztah) [inline docs/project/set-up-dev-env.md]: that *means* there were no dangling.. ?

**Comment by HackToday** [comment]
@thaJeztah yes, it is about this command run

 docker rmi -f $(docker images -q -a -f dangling=true)


as we discussed before, if you get such error, it means , we not have any dangling images

**Comment by thaJeztah** [comment]
@HackToday sorry for the confusion; I meant that the word "means" was missing in that sentence. :smile:

**Comment by HackToday** [comment]
@thaJeztah  PTAL. Thanks

**Comment by thaJeztah** [comment]
thanks @HackToday, LGTM

**Pull request merged**